io/pci/EEH.py

```diff
@@ -105,11 +106,19 @@ def test_eeh_basic_pe(self):
         enter_loop = True
         num_of_miss = 0
         num_of_hit = 0
-        while num_of_hit <= self.max_freeze:
+        while num_of_hit < self.max_freeze:
             for func in self.function:
                 self.log.info("Running error inject on pe %s function %s",
                               self.pci_device, func)
                 if num_of_miss < 5:
+                    if self.pci_class_name == 'scsi_host':
+                        paths_before_eeh = self.get_paths(self.pci_device)
+                        host_state_before_eeh = self.get_host_state(
+                                self.pci_device)
+                        self.log.info("Paths before EEH: %s", 
+                                      paths_before_eeh)
+                        self.log.info("hosts state before EEH: %s", 
+                                      host_state_before_eeh)
```

Comment:
I think this has to collected in the setup itself.
why after missing 5 times. move this to setup.
like if 
```


if self.pci_class_name == 'scsi_host':
                        paths_before_eeh = self.get_paths(self.pci_device)
                        host_state_before_eeh = self.get_host_state(
                                self.pci_device)
                        self.log.info("Paths before EEH: %s", 
                                      paths_before_eeh)
                        self.log.info("hosts state before EEH: %s", 
                                      host_state_before_eeh)


```

Reply:
Added it after every EEH hit as we need to validate paths after every EEH Hit.